pkg/platform/provider/baremetal/phases/keepalived/keepalived.go

```diff
@@ -161,23 +161,23 @@ func installLoadBalanceInIpvsModeIfKubeProxyRunning(s ssh.Interface, option *Opt
 		return nil
 	}
 
-	cmd = fmt.Sprintf("iptables -t nat -C %s -d %s -p tcp --dport 6443 -j DNAT --to-destination %s:443",
+	cmd = fmt.Sprintf("iptables -w 30 -t nat -C %s -d %s -p tcp --dport 6443 -j DNAT --to-destination %s:443",
 		chain, option.VIP, option.KubernetesSvcIP)
 	_, err = s.CombinedOutput(cmd)
-	if err != nil {
-		cmd := fmt.Sprintf("iptables -t nat -I %s -d %s -p tcp --dport 6443 -j DNAT --to-destination %s:443",
+	if err != nil && strings.Contains(err.Error(), "rule exist") {
```

Comment:
Checking if err is caused by xtables lock by string matching, is this solution portable?

Reply:
thanks for your comments, currently we only consider the English OS